NOTICE.html

```diff
@@ -1373,31 +1373,6 @@ <h1>NOTICES AND INFORMATION</h1>
 
 3. Neither the name of Salesforce.com nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
```

Comment:
We can't change the NOTICE.html in the same PR that we change package-lock.json because their content is directly linked. I would recommend you do `npm audit fix` as it's own PR, merge that to main, wait for the main build to pass, and then re-generate this file